app/api/dao/user.py

```diff
@@ -142,20 +143,33 @@ def get_user_by_username(username):
         return UserModel.find_by_username(username)
 
     @staticmethod
-    def list_users(user_id, search_query="", is_verified=None):
+    def list_users(user_id, search_query='', is_verified=None, name_asc='', name_desc=''):
         """ Retrieves a list of verified users with the specified ID.
         
         Arguments:
             user_id: The ID of the user to be listed.
             search_query: The search query for name of the users to be found.
             is_verified: Status of the user's verification; None when provided as an argument.
+            name_asc: 1 for sort ascending order by name.
+            name_desc: 1 for sort descending order by name.
         
         Returns:
             A list of users matching conditions and the HTTP response code.
         
         """
 
-        users_list = UserModel.query.filter(UserModel.id != user_id).all()
+        if name_asc!="" or name_desc!="":
```

Comment:
Having one variable for doing this would reduce the complexity of the below logic.